docs/docs/developer-guides/cosmwasm-cli.md

```diff
@@ -14,33 +14,22 @@ Nix is a requirement to set up and start a local development environment with Co
 
 ### Installing `ccw`
 
-There are two methods to installing the `ccw-vm`:
-
-1. Install the `ccw` crate from the Composable monorepo:
+To install the `ccw-vm`, run the following command using Nix:
 
 ```
-cargo install --git https://github.com/ComposableFi/composable ./composable/code/parachain/frame/cosmwasm/cli
+nix run github:ComposableFi/composable/<insert latest release tag>#ccw --allow-import-from-derivation --extra-experimental-features "flakes nix-command" --no-sandbox --accept-flake-config --option sandbox relaxed
```

Comment:
tbh,  better put tag here and never update.  that is what every does. tag for cli and for devnet. 

people who stick will update anyway.

Reply:
edited